lib/snail.rb

```diff
@@ -35,6 +35,7 @@ class UnknownAttribute < ArgumentError; end
     :locality   => :city,
     :state      => :region,
     :province   => :region,
+    :county     => :region,
```

Comment:
http://www.upu.int/fileadmin/documentsFiles/activities/addressingUnit/irlEn.pdf

^ format for irish mailing addresses.